mmv1/third_party/terraform/utils/dataproc_metastore_service_diff_supress.go.erb

```diff
@@ -0,0 +1,26 @@
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const dataprocMetastoreProvidedOverride = "hive.metastore.warehouse.dir"
+
+func dataprocMetastoreServiceOverrideSuppress(k, old, new string, d *schema.ResourceData) bool {
+	// Suppress diffs for the label provided by Google
+	if strings.Contains(k, dataprocMetastoreProvidedOverride) && new == "" {
```

Comment:
This will make us unable to set the key above, is there ever a reason the user would want to specify that?

Reply:
sync up offline on this?